scripts/travis/build.sh

```diff
@@ -108,6 +108,18 @@ python ${UTILS}/save-logs.py "heron_test_flaky.txt" bazel\
   eco-storm-examples/... eco-heron-examples/...
 end_timer "$T"
 
+# run valgrind tests on cpp tests
+# excluding tuple-cache_unittest because its results are flaky when running under valgrind
+T="heron valgrind test"
+start_timer "$T"
+python ${UTILS}/save-logs.py "heron_valgrind_test.txt" bazel\
+  --bazelrc=tools/travis/bazel.rc test\
+  --run_under=valgrind\
+  --test_summary=detailed --test_output=errors\
+  --config=$PLATFORM --jobs=0\
+  $(bazel query 'kind("cc_test rule", ...)' | grep -v //heron/stmgr/tests/cpp/util:tuple-cache_unittest)
```

Comment:
tmaster is native code as well.

Reply:
`bazel query 'kind("cc_test rule", ...)'` basically returns all test targets that are cpp. tmaster tests are included as well